packages/preview/codly/0.1.0/example/Phos.sublime-syntax

```diff
@@ -0,0 +1,1569 @@
+%YAML 1.2
```

Comment:
This file is probably not necessary for a general purpose package.

Reply:
This is part of the excluded files but sure, I can remove it.

Reply:
I'll let @laurmaedje decide, this most packages include their examples in this repo.

Reply:
You can keep it if you want, but I don't think it's really necessary to demonstrate the package since there are enough built-in languages.